src/components/filters/search-select.tsx

```diff
@@ -62,7 +65,13 @@ export const SearchSelect: React.FC<SearchSelectProps> = ({
     getOptions();
   }, []);
 
-  const optionComponents = optionsData
+  const optionComponents = error
+    ? [
+        <SelectOption key="error" value={ERROR_VALUE}>
+          <Alert variant="danger" isInline isPlain title={error.message || String(error)} />
+        </SelectOption>,
+      ]
+    : optionsData
     ? optionsData.map((item) => (
```

Comment:
I think this nested ternary operator is hard to read because the whitespace doesn't reflect the nesting.

Reply:
Thnx, I refactored this as a function.